Various technical notes/world_infos.json

```diff
@@ -618,7 +618,7 @@
       "asset_id": null,
       "asset_crc32": "E0F1 09CC",
       "asset_name": "0-quentin",
-      "area_name": "Beta characters (IDEM)"
+      "area_name": "⚠ Beta characters (IDEM)"
```

Comment:
This level is impossible to navigate w/o infinite jumps?

Reply:
If you start this level you will instantly fall into the void. All other levels with the ⚠ symbol next to it's name share this behaviour, so I thought it was only logical to include this symbol with this level as well.

Reply:
Yeah that's what it means. Must've missed it then

Reply:
Yeah that's what it means. Must've missed it then. (I sanity checked and you totally fall in the void). Good catch.